### PR TITLE
Compatibility change for KerasNLP 0.4 release

### PR DESCRIPTION
**Pull request by chenmoneygithub**

### CODE REVIEW

File: examples/nlp/fnet_classification_with_keras_nlp.py [hunk 1]
```diff
@@ -160,19 +158,11 @@
 
 
 def train_word_piece(ds, vocab_size, reserved_tokens):
```
Comment (mattdangerw) [inline examples/nlp/fnet_classification_with_keras_nlp.py:160]: why not just remove this whole function and just inline?

also, above this in the paragraph replace

"Luckily, TensorFlow Text
makes it very simple to train WordPiece on a corpus as described in
[this guide](https://www.tensorflow.org/text/guide/subwords_tokenizer)."

with

"We can train a WordPiece vocabulary ourselves with `keras_nlp.tokenizers.compute_word_piece_vocabulary`."

File: examples/nlp/neural_machine_translation_with_keras_nlp.py [hunk 2]
```diff
@@ -146,18 +149,11 @@
```
Comment (mattdangerw) [inline examples/nlp/neural_machine_translation_with_keras_nlp.py:149]: Same here, above this line, replace the "Luckily" sentence with the suggestion I had above or something like it.

File: examples/nlp/neural_machine_translation_with_keras_nlp.py [hunk 3]
```diff
@@ -449,70 +455,6 @@ def token_probability_fn(decoder_input_tokens):
     print(translated)
     print()
 
-"""
-After 10 epochs, we get samples like these:
-
-** Example 0 **
-
-have you seen this?
```
Comment (mattdangerw) [inline examples/nlp/neural_machine_translation_with_keras_nlp.py:457]: should we keep a few examples in? Will help readability when not running this as a colab.

File: guides/keras_nlp/transformer_pretraining.py [hunk 4]
```diff
@@ -217,7 +217,7 @@
 requireing a massive vocabulary of every word in our training set.
 
 The second thing we need to do is mask our input for the MLM task. To do this, we can use
```
Comment (mattdangerw) [inline guides/keras_nlp/transformer_pretraining.py:219]: MLM -> MaskedLM

And rename this section "Preprocess data for the MLM task" -> "Preprocess data for the MaskedLM task"

File: guides/keras_nlp/transformer_pretraining.py [hunk 5]
```diff
@@ -338,7 +338,7 @@ def preprocess(inputs):
 You can think of the `encoder_model` as it's own modular unit, it is the piece of our
 model that we are really interested in for our downstream task. However we still need to
 set up the encoder to train on the MLM task; to do that we attach a
```
Comment (mattdangerw) [inline guides/keras_nlp/transformer_pretraining.py:340]: MLM -> MaskedLM

**Review by mattdangerw — CHANGES_REQUESTED** [review]
Looks good! Just a few minor comments.

File: guides/md/keras_nlp/transformer_pretraining.md [hunk 6]
```diff
@@ -146,6 +159,9 @@ print(sst_train_ds.unbatch().batch(4).take(1).get_single_element())
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:From /opt/conda/envs/myenv/lib/python3.9/site-packages/tensorflow/python/autograph/pyct/static_analysis/liveness.py:83: Analyzer.lamba_check (from tensorflow.python.autograph.pyct.static_analysis.liveness) is deprecated and will be removed after 2023-09-23.
```
Comment (mattdangerw) [inline guides/md/keras_nlp/transformer_pretraining.md:162]: Can we remove these three lines, this should be fixed in upstream tensorflow shortly, and is just distracting for now.

File: guides/md/keras_nlp/transformer_pretraining.md [hunk 7]
```diff
@@ -636,12 +660,329 @@ print(restored_model(inference_data))
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.
```
Comment (mattdangerw) [inline guides/md/keras_nlp/transformer_pretraining.md:663]: Can we remove this large block of WARNING and INFO logs for now? Not sure what's going on here.

**Review by mattdangerw — APPROVED** [review]
LGTM, just a few spurious warnings we might want to avoid rendering on keras.io

**Comment by fchollet** [comment]
It seems there are commits from `ch*****ey​@chenmoney-gpu.c.keras-team-gcp.internal` which does not have a CLA associated with it. Please create a single commit with the changes to fix this issue.

**Comment by chenmoneygithub** [comment]
@fchollet Commits from GCP compute engine all have this weird CLA issue, and I cannot find a way to sign it. Is this issue blocking? I need to run the generation script from GCP because it takes a few hours on my outdated GPU. 

If this is blocking, maybe we can do the review and address comments first, then squashing all commits outside GCP, so I don't need to run the generation script?

**Comment by fchollet** [comment]
It seems I'm actually able to merge without the CLA. Let me try that.

Otherwise the fix would have been to do a clean commit locally. This can be done after committing files from the GCP instance.

> Commits from GCP compute engine all have this weird CLA issue

In the future, all you need is to configure `git` locally on the GCP instance so that it uses your regular email address. You can do so via `git config --global user.email "chenmoney@google.com"`

**Pull request merged**